RELEASE/scripts/autoscend/auto_cooking.ash

```diff
@@ -773,6 +814,10 @@ boolean loadConsumables(string _type, ConsumeAction[int] actions)
 	{
 		use(item_amount($item[van key]), $item[van key]);
 	}
+	if ((item_amount($item[Knob Goblin lunchbox]) > 0) && (pulls_remaining() != -1))
+	{
+		use(item_amount($item[Knob Goblin lunchbox]), $item[Knob Goblin lunchbox]);
+	}
```

Comment:
```suggestion
	boolean missing_lunchbox_content = item_amount($item[knob pasty]) == 0 || item_amount($item[thermos full of Knob coffee]) == 0;
	if (item_amount($item[Knob Goblin lunchbox]) > 0 && (pulls_remaining() != -1 || missing_lunchbox_content))
	{
		use(item_amount($item[Knob Goblin lunchbox]), $item[Knob Goblin lunchbox]);
	}
```

that will only open a single lunchbox. and will only open more if we finish up all the food or drink it provides. thus allowing its use in casual without opening all of them.

Reply:
i think this is unnecessary

Reply:
unless you have a lot of IOTMs then this can be the best food or drink available in run for casual. Especially if we add the option to do get all the semirares for food and drink (which I plan on doing since it is worth it for low IOTM accounts). Aside from this, this also ensures we do not wastefully open more lunchboxes than we need when not in casual, instead opening them one at a time and only if we ran out of the contents.

Reply:
Also unlike duffel bag and van key (whose contents are random), here the contents are known in advance.

Reply:
done. edited my suggestion.
With this regardless of casual or non casual, it will check the amount of knob pasty and thermos full of knob coffee. and only if we are completely out of one of them would it open a single lunchbox.
That stops it from wasting lunchboxes in standard too, only opening as many as it needs.

Reply:
I really don't agree with that since in casual you have full access to the mall and aren't under standard restrictions.

Reply:
Mallbuying costs money which we might not have... hmm, it would be more efficient to sell the lunchbox though and buy an elemental caiproska though and a dinner roll. so ok, nevermind it would be a mistake in casual after all.

But what about not in casual? should we open the boxes only if out of contents in standard to preserve extra boxes that we do not need?

Reply:
Also (pulls_remaining() != -1) is a rather unclear way to indicate we got mall access. if that was the goal then it should be can_interact()

Reply:
Alright. changed suggestion to:
`if (item_amount($item[Knob Goblin lunchbox]) > 0 && !can_interact() && missing_lunchbox_content)`
!can_interact() means that we do not have unrestricted mall access. such as casual or postronin.
in ronin it will open them 1 at a time. thoughts?

Reply:
wait... does postronin actually alleviate standard restrictions in a path?

Reply:
well regardless it is a pretty trivial change for standard. so its fine to not include it